atomic_reactor/util.py

```diff
@@ -862,3 +866,49 @@ def are_plugins_in_order(plugins_conf, *plugins_names):
         except ValueError:
             return False
     return True
+
+
+def read_yaml(yaml_file_path, default=None, schema=None):
+    with open(yaml_file_path) as f:
+        data = yaml.safe_load(f)
+
+    if not schema:
+        return data
```

Comment:
Hmm I guess not. I'll add a unit test for it.
I still think there's value in keep the functionality though.